app/src/amazonWebview/java/org/mozilla/focus/web/WebViewProvider.java

```diff
@@ -39,26 +39,6 @@ public static void performCleanup(final Context context) {
         SystemWebView.deleteContentFromKnownLocations(context);
     }
 
-    /**
-     * A cleanup that should occur when a new browser session starts. This might be able to be merged with
-     * {@link #performCleanup(Context)}, but I didn't want to do it now to avoid unforeseen side effects. We can do this
-     * when we rethink our erase strategy: #1472.
-     *
-     * This function must be called before WebView.loadUrl to avoid erasing current session data.
-     */
-    public static void performNewBrowserSessionCleanup() {
```

Comment:
Do we need to do this for "clear browser history"? It might be okay (e.g., bc we don't care about it) (but I remember Seburo mentioning something to barbara in an email, but I don't know what he was referencing exactly).

Reply:
- No, this is only for cleanup when starting a new browsing session
- Seburo was mentioning the issue about CSS link coloring not clearing after we delete history: https://github.com/mozilla-mobile/focus-android/issues/828